app/views/forms/odk_manifest.xml.erb

```diff
@@ -7,5 +7,15 @@
         <downloadUrl><%= url_for_path(@ifa.path) %></downloadUrl>
       </mediaFile>
     <% end %>
+
+    <% unless @decorated_questions.empty? %>
+      <% @decorated_questions.each do |question| %>
+        <mediaFile>
+          <filename><%= question.unique_audio_prompt_filename %></filename>
+          <hash><%= question.audio_prompt_md5 %></hash>
+          <downloadUrl><%= url_for_path(audio_prompt_question_path(question)) %></downloadUrl>
```

Comment:
I think you can just use `audio_prompt_question_url(question)`.

Remember, when pasting code, always scrutinize every piece -- is this still relevant? Do I need this? Does it work in this context?

Reply:
I didn't try it without the url_for_path helper but didn't just add it. I checked the method and noticed it worked with the http(s) protocols. That's why I used it but using the `_url` helper is...nicer